pkg/exec/command.go

```diff
@@ -6,7 +6,7 @@ import (
 	"os/exec"
 	"time"
 
-	"github.com/contiv/executor"
+	"github.com/idealhack/executor"
```

Comment:
I'm afraid I will refuse about this. Do you have optimized this package? If yes, can you add it in our `pkg`